tools/star.wdl

```diff
@@ -585,20 +585,20 @@ task alignment {
         Int alignWindowsPerReadNmax = 10000
         Int alignTranscriptsPerWindowNmax = 100
         Int alignTranscriptsPerReadNmax = 10000
-        Int peOverlapNbasesMin = 0
+        Int peOverlapNbasesMin = 10
         Int winAnchorMultimapNmax = 50
         Int winBinNbits = 16
         Int winAnchorDistNbins = 9
         Int winFlankNbins = 4
-        Int chimSegmentMin = 0
+        Int chimSegmentMin = 10
         Int chimScoreMin = 0
-        Int chimScoreDropMax = 20
-        Int chimScoreSeparation = 10
-        Int chimScoreJunctionNonGTAG = -1
-        Int chimJunctionOverhangMin = 20
-        Int chimSegmentReadGapMax = 0
+        Int chimScoreDropMax = 30
```

Comment:
I stopped reviewing here in the list. My main thought is this: should we stick to the STAR defaults for the WDL task defaults and then only break these out for our Arriba-related workflows? Let me know what you think of that.

Reply:
I think that's reasonable. The downside, and this is a failing of WDL in my opinion, is that overriding at the workflow level is cumbersome. If you override in the `call` block, then the user is blocked from changing the value, even if the workflow otherwise enables `allowNestedInputs`. So you're forced in to explicitly declaring `inputs` at the workflow level for these STAR parameters. We can do that, but it is significantly more cluttered.